qiling/core.py

```diff
@@ -133,9 +133,11 @@ def __init__(
         if self.log_dir != "" and type(self.log_dir) == str:
             _logger = ql_setup_logging_env(self)    
             self.log_file_fd = _logger
-        else:
-            self.log_dir = None     
-        
+        elif self.console == True:
```

Comment:
this only work during startup? what if we want to turn on or off console during execution or within a hook?

Reply:
add a simple console checking will let you do it, turn off or on console logging by `ql.console = True` or `ql.console = False` , is this enough or we need more granular ?

Reply:
like this
```python
import sys
sys.path.append("..")
from qiling.os.posix.syscall.fcntl import ql_syscall_openat
from qiling import *

def my_openat(ql, *args):
    ql.console = False
    ql_syscall_openat(ql, *args)
    ql.console = True


if __name__ == "__main__":
    ql = Qiling(["examples/rootfs/x8664_linux/bin/x8664_hello"], "examples/rootfs/x8664_linux", console=True)
    ql.set_syscall("openat", my_openat)
    ql.run()

```

Reply:
can you test it this way ?
```
def my_openat(ql, *args):
    ql.console = False
    ql,nprint("Shows nothing") 
    ql.console = True
    ql,nprint("Shows something")
```

Reply:
![image](https://user-images.githubusercontent.com/10076774/82110080-36cfb480-976e-11ea-8942-c8266d062beb.png)

Reply:
Nice